docs/index.rst

```diff
@@ -329,30 +335,6 @@ for passwords.
 See `twine 338 <https://github.com/pypa/twine/issues/338>`_ for
 discussion and background.
 
-Resources
----------
-
-* `GitHub repository <https://github.com/pypa/twine>`_
-* User and developer `documentation`_
-* `Python Packaging User Guide`_
-* `Python packaging issue tracker <https://github.com/pypa/packaging-problems>`_
-* `IRC <https://web.libera.chat/#pypa>`_:
-  ``#pypa`` on irc.libera.chat
```

Comment:
It's feels like these could be useful in readme

Reply:
Maybe, although my preference would be to direct visitors to the docs, rather than build out the readme.

BTW, after working on this and #792, I'm reconsidering my stance on including `README.rst`  in `index.rst` (https://github.com/pypa/twine/pull/784#issuecomment-885611297). I think it makes more sense if _all_ the `README` content fits as the `index` intro. Assuming this is merged, I'll probably take a pass at that.

Reply:
> Maybe, although my preference would be to direct visitors to the docs, rather than build out the readme.

I often take into account it's something that is also shown in other places like PyPI and GH, and so I want to include direct links to places w/o having to go through a proxy page (the index document in docs).

> BTW, after working on this and #792, I'm reconsidering my stance on including `README.rst` in `index.rst` ([#784 (comment)](https://github.com/pypa/twine/pull/784#issuecomment-885611297)). I think it makes more sense if _all_ the `README` content fits as the `index` intro. Assuming this is merged, I'll probably take a pass at that.

This is what I mostly do. One note: I usually still exclude the docs badge because it doesn't make sense to have it in sphinx. Also, it wouldn't hurt to have a regular rst linter on documents like readme to avoid accidentally using sphinx-only roles and directives. Same applies to other pages (like contributing/security/CoC that are show both on GitHub and Sphinx).

Reply:
> I often take into account it's something that is also shown in other places like PyPI and GH

Good point, although for [PyPI](https://pypi.org/project/twine/), I might lean towards the "Project Links" (i.e. `metadata.project_urls`), which already includes PyPUG.

> it wouldn't hurt to have a regular rst linter

Looks like we're already running [doc8](https://github.com/pycqa/doc8). Is there another that you recommend, or a configuration that we should use?

https://github.com/pypa/twine/blob/84a5d3c6c7edf29b8ff489166325c493cca08691/tox.ini#L37